server/schedule/region_scatterer_test.go

```diff
@@ -432,3 +433,37 @@ func (s *testScatterRegionSuite) TestSelectedStoreGC(c *C) {
 	_, ok = stores.getGroupDistribution("testgroup")
 	c.Assert(ok, Equals, false)
 }
+
+func (s *testScatterRegionSuite) TestRegionFromDifferentGroups(c *C) {
```

Comment:
would you like to add more comments for this test?

Reply:
updated.